openslides_backend/presenter/check_database.py

```diff
@@ -0,0 +1,78 @@
+from typing import Any, Dict, List
+
+import fastjsonschema
+from datastore.shared.util import DeletedModelsBehaviour
+
+from ..action.actions.meeting.export_helper import export_meeting
+from ..models.checker import Checker, CheckException
+from ..permissions.management_levels import OrganizationManagementLevel
+from ..permissions.permission_helper import has_organization_management_level
+from ..shared.exceptions import PermissionDenied
+from ..shared.schema import schema_version
+from .base import BasePresenter
+from .presenter import register_presenter
+
+check_database_schema = fastjsonschema.compile(
+    {
+        "$schema": schema_version,
+        "type": "object",
+        "title": "check database",
+        "description": "check database",
+        "properties": {},
+    }
+)
+
+
+@register_presenter("check_database")
+class CheckDatabase(BasePresenter):
+    """Check Database gets all non-deleted meetings, exports them,
+    and check them with the checker."""
+
+    schema = check_database_schema
+
+    def get_result(self) -> Any:
+        # check permissions
+        if not has_organization_management_level(
+            self.datastore, self.user_id, OrganizationManagementLevel.SUPERADMIN
+        ):
+            msg = "You are not allowed to perform presenter check_database."
+            msg += f" Missing permission: {OrganizationManagementLevel.SUPERADMIN}"
+            raise PermissionDenied(msg)
+
+        meeting_ids = self.get_all_meeting_ids()
+        errors: Dict[int, str] = {}
+        for meeting_id in meeting_ids:
+            export = export_meeting(self.datastore, meeting_id)
+            checker = Checker(
+                data=export,
+                mode="internal",
+                repair=True,
```

Comment:
IMO should be False. The `all_*`-fields are not checked and the others doesn't matter, if they are on the errors list. I'm not quite sure, but internal should ignore relations to the other meetings.
But we want the missing default values as errors.

Reply:
If repair is False, the checker will complain, that a lot of fields are missing, every field must be in the meeting, etc. This is not correct.

Reply:
Set the repair to False, and update the correct test. I added the default fields.